### PR TITLE
Remove unused components

### PR DESCRIPTION
**Pull request by justin0022**
Addresses https://github.com/tl-its-umich-edu/my-learning-analytics/issues/428

Left in some unused components that I thought would be used in the near future, like `Scatterplot`, `LineChart`, and `EmojiFeedback`. 

All removed components can be found in [this repo](https://github.com/justin0022/dashboard), in case we need them in the far future.

### CODE REVIEW

File: assets/src/components/GradeDistribution.js [hunk 1]
```diff
@@ -1,9 +0,0 @@
-import createHistogram from './d3/createHistogram'
-import withResponsiveness from './withResponsiveness'
-import createChartComponent from './createChartComponent'
-import compose from '../util/compose'
-
-export default compose(
-  withResponsiveness,
-  createChartComponent
-)(createHistogram)
```
Comment (pushyamig) [inline assets/src/components/GradeDistribution.js]: I thought this file is being used for Grades view?

**Comment by pushyamig** [comment]
You can leave the `FileAccessed.js` I know it will be used. But other files are not relevant so it would be a distraction.

File: assets/src/components/GradeDistribution.js (hunk 1 above)
Comment (justin0022) [reply assets/src/components/GradeDistribution.js]: The file name is misleading, it isn't being used. 

The dependencies for Grade Distribution view are:
```javascript
import React from 'react'
import { renderToString } from 'react-dom/server'
import useFetch from '../hooks/useFetch'
import { withStyles } from '@material-ui/core/styles'
import Paper from '@material-ui/core/Paper'
import Grid from '@material-ui/core/Grid'
import Typography from '@material-ui/core/Typography'
import Histogram from '../components/Histogram'
import Spinner from '../components/Spinner'
import createToolTip from '../util/createToolTip'
import Table from '../components/Table'
import { average } from '../util/math'
```

**Comment by justin0022** [comment]
> You can leave the `FileAccessed.js` I know it will be used. But other files are not relevant so it would be a distraction.

Done!

**Pull request merged**